ProjectLighthouse.Servers.Website/Extensions/FormattingExtensions.cs

```diff
@@ -5,7 +5,41 @@ namespace LBPUnion.ProjectLighthouse.Servers.Website.Extensions;
 
 public static class FormattingExtensions
 {
-    public static string GetLevelLockIcon(this SlotEntity slot) => slot.InitiallyLocked ? "icon lock" : "";
+    private static string GetLevelLockIcon(this SlotEntity slot) => slot.InitiallyLocked ? "ui black icon lock" : "";
+    private static string GetTeamPickedIcon(this SlotEntity slot) => slot.TeamPick ? "ui pink icon certificate" : "";
+
+    // ReSharper disable once UnusedParameter.Global
+    private static string GetLevelWarningIcon
+        (this SlotEntity slot) =>
+        slot.Lbp1Only || slot.CrossControllerRequired || slot.MoveRequired ? "ui orange icon exclamation circle" : "";
+
+    // These messages are sorted by logical priority. No two should happen at once.
+    private static string GetLevelWarningText(this SlotEntity slot)
+    {
+        return slot switch
+        {
+            {
+                Lbp1Only: true,
+            } => "This level is designed for LittleBigPlanet 1 only.",
+            {
+                CrossControllerRequired: true,
+            } => "This level is designed to be played in Cross-Controller mode using a PlayStation Vita system.",
+            {
+                MoveRequired: true,
+            } => "This level requires a PlayStation Move controller.",
+            _ => "",
+        };
```

Comment:
```suggestion
        {
            {
                Lbp1Only: true,
            } => "This level can be played via LBP 1 only.",
            {
                CrossControllerRequired: true,
            } => "This level requires the LBP2: Cross-Controller DLC and companion app.",
            {
                MoveRequired: true,
            } => "This level requires a PlayStation Move controller.",
            _ => "",
        };
```
you dont need to follow this directly but we should make it a bit more brief and remove direct references to Sony's trademarks.